cmd/mothership/mothership/start/http.go

```diff
@@ -110,8 +111,12 @@ func startWebserver(ctx context.Context, o *Options) error {
 		fmt.Sprintf("/v{%s}/clusters/{%s}/config/{%s}", paramContractVersion, paramRuntimeID, paramConfigVersion),
 		callHandler(o, getKymaConfig)).Methods(http.MethodGet)
 
+	apiRouter.HandleFunc(
+		fmt.Sprintf("/v{%s}/occupancy/{%s}", paramContractVersion, paramPoolID),
```

Comment:
Here, the endpoint is called using only the http delete method:
```
apiRouter.HandleFunc(
		fmt.Sprintf("/v{%s}/occupancy/{%s}", paramContractVersion, paramPoolID),
		callHandler(o, deleteComponentWorkerPoolOccupancy)).Methods(http.MethodDelete)
```
 Which is visible enough because you need to be intentional by using the http delete verb when you call it. If we add another endpoint to get it, it would have the same path but supports only the get http verb like so:
 ```
apiRouter.HandleFunc(
		fmt.Sprintf("/v{%s}/occupancy/{%s}", paramContractVersion, paramPoolID),
		callHandler(o, getComponentWorkerPoolOccupancy)).Methods(http.MethodGet)
```